### ISSUE TITLE
warning " > eslint-plugin-rebase@0.1.0" has incorrect peer dependency "eslint@^6.8.0".

### ISSUE THREAD

**Issue by binary64** [AndersDJohnson/eslint-plugin-rebase#46]
I get this when `yarn install`ing:-

```
warning " > eslint-plugin-rebase@0.1.0" has incorrect peer dependency "eslint@^6.8.0".
```

Your plugin seems to work on 7.30.0 just fine!

**Comment by AndersDJohnson** [comment]
@binary64 Thanks! Fixed in https://github.com/AndersDJohnson/eslint-plugin-rebase/pull/47, will publish soon!

**Comment by AndersDJohnson** [comment]
@binary64 Published with v0.1.1.
